src/main/java/de/tum/in/www1/artemis/service/connectors/gitlab/dto/GitLabPersonalAccessTokenRequestDTO.java

```diff
@@ -8,59 +8,7 @@
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-public class GitLabPersonalAccessTokenRequestDTO {
+public record GitLabPersonalAccessTokenRequestDTO(@JsonProperty String name, @JsonProperty("user_id") Long userId, @JsonProperty("scopes") String[] scopes,
```

Comment:
Same here - why do we need `@JsonProperty` for name?

Reply:
might be possible to leave it out, we just took it over. we would need to test that it does not break